integreat_cms/cms/models/media/media_file.py

```diff
@@ -202,6 +204,48 @@ def thumbnail_url(self):
             f"{settings.BASE_URL}{self.thumbnail.url}?{self.last_modified.timestamp()}"
         )
 
+    @property
+    def used_in(self):
+        """
+        Return where the media file is used.
+
+        :return: name and model of the contents in which the media file is used.
+        :rtype: str
+        """
+
+        as_icon = (
+            list(self.icon_organizations.all())
+            + list(self.icon_regions.all())
+            + list(self.events.all())
+            + list(self.pages.all())
+            + list(self.pois.all())
+        )
+
+        as_content = []
+        try:
+            url = Url.objects.get(url=self.url)
+            as_content = list(url.links.all())
+        except Url.DoesNotExist:
+            as_content = []
```

Comment:
```suggestion
        as_content = list(Link.objects.filter(url__url=self.url))
```
You can also go directly via the link model.
But we really have to think about performance here, since it's impossible to prefetch all link objects for the media elements since `url` is a property of the media model and not a database field. Maybe we have to drop this idea :(
Or do you have another idea how to optimize this?